query_modules/example.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <mg_exceptions.hpp>
+#include <mgp.hpp>
```

Comment:
What about moving the exceptions into the `mgp.hpp` because when you include the header you include one thing? That's I would say standard for library code 😄

Reply:
If `mgp.hpp` includes the exceptions file, this will work out of the box, not the best practice but that's I think an ok solution here (just to include exception inside mgp.hpp, if possible) 😄

Reply:
It actually works without. It's all in `mgp.hpp`

Reply:
I will delete the include